samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1701,6 +1712,8 @@ mfxStatus CDecodingPipeline::RunDecoding()
     CTimeInterval<>     decodeTimer(m_bIsCompleteFrame);
     time_t              start_time = time(0);
     std::thread         deliverThread;
+    mfxMemId            inMid = nullptr;
+    mfxMemId            outMid = nullptr;
```

Comment:
why do you need these declarations here? they look to be limited scope ones. Move below to the scope where you actually use them.

Reply:
Sure. I will change it. Thanks.